FreeSurfer/scripts/FreeSurferHiresPial.sh

```diff
@@ -168,12 +179,19 @@ cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.pial $SubjectDIR/$Subjec
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/lh.thickness $SubjectDIR/$SubjectID/surf/lh.thickness.preT2.pass2
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.thickness $SubjectDIR/$SubjectID/surf/rh.thickness.preT2.pass2
 
-# Generate pial surfaces with T2 adjustment, second pass.  
-# Same 4 files created as above, but have $outputSuffix2 ("postT2.pass2") appended through use of -output flag
-#Could go from 3 to 2 potentially...
-outputSuffix2=".postT2.pass2"
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix2 $SubjectID lh
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix2 $SubjectID rh
+
+if [ ! "${T2wImage}" = "NONE" ] ; then
+  verbose_red_echo "---> Generating pial second pass surfaces with T2 adjustment."
+  # Generate pial surfaces with T2 adjustment, second pass.
+  # Same 4 files created as above, but have $outputSuffix2 ("postT2.pass2") appended through use of -output flag
+  #Could go from 3 to 2 potentially...
+  outputSuffix2=".postT2.pass2"
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix2 $SubjectID lh
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix2 $SubjectID rh
+else
+  verbose_red_echo "---> No T2w image, skipping generation of pial second pass surfaces with T2 adjustment."
+  outputSuffix2=""
```

Comment:
For consistency, shouldn't we do something similar then to `outSuffix2`?  e.g.,
`outputSuffix2=preT2.pass2`

Reply:
agreed done.